AudioAnalytics/AudioAnalyzer/src/main/java/sample/MediaServicesEventProcessor.java

```diff
@@ -3,37 +3,60 @@
 
 package sample;
 
-import com.microsoft.azure.eventprocessorhost.IEventProcessor;
-import com.microsoft.azure.eventprocessorhost.PartitionContext;
-import com.microsoft.azure.eventhubs.EventData;
-import com.microsoft.azure.eventprocessorhost.CloseReason;
+import com.azure.messaging.eventhubs.EventData;
+import com.azure.messaging.eventhubs.EventProcessorClient;
+import com.azure.messaging.eventhubs.EventProcessorClientBuilder;
+import com.azure.messaging.eventhubs.checkpointstore.blob.BlobCheckpointStore;
+import com.azure.messaging.eventhubs.models.*;
+import com.azure.storage.blob.BlobContainerAsyncClient;
+import org.json.simple.JSONArray;
 import org.json.simple.JSONObject;
 import org.json.simple.parser.JSONParser;
-import org.json.simple.JSONArray;
+
+import java.util.function.Consumer;
 
 /**
  * Implementation of IEventProcessor to handle events from Event Hub.
  */
-public class MediaServicesEventProcessor implements IEventProcessor {
+public class MediaServicesEventProcessor {
     private final Object MONITOR;
     private final String JOB_NAME;
     private final String LIVE_EVENT_NAME;
+    private final String EVENT_HUB_CONN_STRING;
```

Comment:
This is not standard java naming... should be `monitor`, `jobName`, `liveEventName`, etc.

Reply:
According to your comments, it has been fixed in the new version.